### DISCUSSION TITLE
Welcome to the Digifiz project!

### DISCUSSION THREAD

**Discussion by gfunkbus76** [gfunkbus76/Digifiz-Dash#2] [General]
Hello friends...

Welcome to the Digifiz project. Not a very accomplished coder or github-er so please bear with me. 

Discussions might be best way to contact me regarding this project.

Cheers!

**Comment by oscardurst** [comment]
Hello @gfunkbus76, i just started to analize your code and the application, and looks very similar to the VW digifiz, congratulations for your project!!! i just was wondering how to add more RPM to the application, yours shows 5000 RPM at maximum RPM, how can i add 7000 RPM? different images are needed?? 
My project consists in a GTI MK2 and i would like to add the digifiz cluster.

Thanks in advanced.

**Reply by gfunkbus76** [reply]
Hey Oscar!

Unfortunately, I set this up for my Diesel motor so I adjusted the room at my original design stage.

I could probably change an rpm line(the white line and numbers 1-5 to 1-7 and then each bar on the display would be equal to 50/70 or like 72rpm instead of 100 as default. There is also the glow plug logo I could swap with something else.

Weas thinking of making a bone stock version of the digifiz, as mine adds some gauges.

Note that the MFA is currently only showing ambient temp. Will add functions and corresponding graphics as I can.

**Reply by gfunkbus76** [reply]
Here's what I'm going to put in my bus lol... Mk2 dash modified.

https://youtu.be/TXlMsQnUzd8

**Comment by MilesHodgson** [comment]
Reviving this old thread to say thank you, serving as an excellent base for my own project. Will be putting this in my vw caddy mk1 with the same engine you had (AAZ) and my G60 Corrado.

Using ESP32's to get data from sensors and sending them over MQTT to a broker running on the PI thats running the UI.
